.gitignore

```diff
@@ -3,4 +3,4 @@ Packages
 xcuserdata
 DerivedData/
 Carthage
-CombineExt.framework.zip
\ No newline at end of file
+CombineExt.framework.zip
```

Comment:
I think the `.DS_Store` file should be added to `.gitignore`, but I didn't make that change in this PR. Is there a reason to commit this file?

Reply:
Would be great to add it ! Thanks

Reply:
Done